.github/workflows/run-examples.yml

```diff
@@ -45,7 +42,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: System Tests
-        run: ./mill test.testOnly '*SystemTests*' || true
+        run: ./mill test.testOnly 'SystemTests*' || true
```

Comment:
Now we expect SystemTests to pass we should remove the `|| true`, its possibly also worth adding the AnalysisSystemTests as a separate stage.

Reply:
We don't expect SystemTests to pass yet - arrays_simple doesn't pass yet for reasons unrelated to indirect calls and that has long been the case.

AnalysisSystemTests is not really worth putting in the GitHub action - it's useful to check for crashes but rather redundant in total. Many of the other test suites may be worth including though.